pybv/tests/test_bv_writer.py

```diff
@@ -97,18 +97,22 @@ def test_bv_bad_format(tmpdir):
     with pytest.raises(ValueError, match='Orientation bad not supported'):
         _write_vhdr_file(vhdr_fname, vmrk_fname, eeg_fname, data=data,
                          sfreq=sfreq, ch_names=ch_names, orientation='bad',
-                         format="binary_float32", resolution=1e-6, unit="V")
+                         format="binary_float32", resolution=1e-6,
+                         units=["V"] * n_chans)
     with pytest.raises(ValueError, match='Data format bad not supported'):
         _write_vhdr_file(vhdr_fname, vmrk_fname, eeg_fname, data=data,
                          sfreq=sfreq, ch_names=ch_names,
                          orientation='multiplexed', format="bad",
-                         resolution=1e-6, unit="V")
+                         resolution=1e-6,
+                         units=["V"] * n_chans)
     with pytest.raises(ValueError, match='Orientation bad not supported'):
         _write_bveeg_file(eeg_fname, data=data, orientation='bad',
-                          format="bad", resolution=1e-6, unit="µV")
+                          format="bad", resolution=1e-6,
+                          units=["µV"] * n_chans)
     with pytest.raises(ValueError, match='Data format bad not supported'):
         _write_bveeg_file(eeg_fname, data=data, orientation='multiplexed',
-                          format="bad", resolution=1e-6, unit="µV")
+                          format="bad", resolution=1e-6,
+                          units=["µV"] * n_chans)
```

Comment:
Why these additional arguments? Don't the original test work anymore?

Reply:
Passing in a single string to the private function is not supported anymore (and the tests will fail) because it expects a list of units, which is set in the public API.

Reply:
The public API still supports a single string